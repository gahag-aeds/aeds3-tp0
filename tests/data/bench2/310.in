310 155
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 
Add 1 103
Sum 195 297
Sub 22 124
Max 143 245
Sub 136 238
Max 194 296
Sub 88 190
Min 100 202
Add 163 265
Sum 141 243
Sub 73 175
Sum 9 111
Add 6 108
Sum 72 174
Add 63 165
Sum 153 255
Add 95 197
Sum 165 267
Sub 3 105
Min 184 286
Sub 111 213
Min 40 142
Sub 94 196
Max 65 167
Add 4 106
Min 85 187
Add 22 124
Min 154 256
Sub 194 296
Max 65 167
Add 59 161
Sum 165 267
Add 53 155
Min 52 154
Sub 101 203
Sum 159 261
Sub 9 111
Max 188 290
Add 51 153
Min 142 244
Add 92 194
Min 94 196
Add 150 252
Min 97 199
Add 136 238
Min 184 286
Sub 42 144
Sum 201 303
Add 141 243
Max 44 146
Sub 178 280
Max 44 146
Add 54 156
Min 64 166
Sub 79 181
Sum 57 159
Sub 119 221
Max 71 173
Add 181 283
Sum 105 207
Add 136 238
Min 4 106
Add 34 136
Sum 159 261
Sub 55 157
Max 134 236
Sub 110 212
Sum 63 165
Sub 85 187
Max 161 263
Add 205 307
Max 176 278
Sub 18 120
Min 118 220
Sub 36 138
Min 154 256
Sub 65 167
Max 77 179
Add 147 249
Min 205 307
Add 91 193
Min 88 190
Sub 13 115
Sum 144 246
Add 123 225
Max 62 164
Add 171 273
Min 34 136
Add 133 235
Min 115 217
Add 110 212
Max 32 134
Sub 118 220
Sum 114 216
Add 190 292
Max 143 245
Sub 40 142
Sum 63 165
Sub 177 279
Max 95 197
Sub 194 296
Min 145 247
Sub 34 136
Max 5 107
Sub 26 128
Sum 69 171
Sub 129 231
Min 91 193
Add 116 218
Sum 39 141
Add 88 190
Max 206 308
Sub 98 200
Max 71 173
Add 113 215
Min 13 115
Add 133 235
Max 47 149
Add 17 119
Max 29 131
Add 132 234
Min 97 199
Add 60 162
Max 14 116
Add 3 105
Min 13 115
Add 156 258
Min 13 115
Sub 46 148
Sum 96 198
Sub 78 180
Sum 33 135
Add 33 135
Sum 111 213
Add 8 110
Max 59 161
Add 150 252
Min 136 238
Sub 120 222
Min 58 160
Add 145 247
Sum 207 309
Add 27 129
Max 35 137
Sub 42 144
Max 110 212
Add 47 149
Sum 174 276
Add 127 229
Max 35 137
Sub 173 275
Min 52 154
Sub 131 233
Max 203 305
Sub 65 167
