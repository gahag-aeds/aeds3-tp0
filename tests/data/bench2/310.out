5242
99
98
-2
5300
4975
4715
5542
5670
0
-1
99
-2
-2
101
5515
3
5488
98
-3
-2
-1
-1
5177
102
102
2
4955
102
5045
1
5774
99
4883
99
100
-3
-3
99
0
-2
5523
98
-3
2
98
5377
105
4612
100
1
99
4541
0
4542
102
100
-5
95
96
1
100
-2
-2
5445
4633
5486
98
5
-2
5614
99
104
6065
99
-1
105
